modules/pipeline/conf/conf.go

```diff
@@ -111,6 +111,9 @@ type Conf struct {
 
 	// k8s executor goroutine pool size
 	K8SExecutorPoolSize int `env:"K8S_EXECUTOR_POOL_SIZE" default:"50"`
+
+	// external market refresh interval
+	ExtensionVersionRefreshIntervalMinute uint64 `env:"EXTENSION_VERSION_REFRESH_INTERVAL_MINUTE" default:"20"`
```

Comment:
dicehub need push event like clusterinfo.

Reply:
// todo

Reply:
default 20 is too long. i think 1min is better before we can listen ext change events.